### PR TITLE
feat(qwik): add 6 more examples for Qwik

### PR DESCRIPTION
**Pull request by SimoMay**

### CODE REVIEW

**Review by mhevery — APPROVED** [review]

File: content/2-templating/2-styling/qwik/CssStyle.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { component$, Host, useStyles$ } from '@builder.io/qwik';
+
+export const AppCss = `
```
Comment (matschik) [inline content/2-templating/2-styling/qwik/CssStyle.tsx:3]: Is style export worth it ?

File: content/2-templating/4-event-click/qwik/Counter.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { component$, useStore, Host } from '@builder.io/qwik';
+
+export const Counter = component$(() => {
+	const store = useStore({ count: 0 });
+	return (
+  <Host>
+    <p>Counter: {store.count}</p>
+    <button onClick$={() => store.count++}>Increment</button>
+  </Host>
+  );
```
Comment (matschik) [inline content/2-templating/4-event-click/qwik/Counter.tsx:10]: Missing formatting

**Review by matschik — CHANGES_REQUESTED** [review]

File: content/2-templating/2-styling/qwik/CssStyle.tsx (hunk 1 above)
Comment (SimoMay) [reply content/2-templating/2-styling/qwik/CssStyle.tsx]: fixed!

**Comment by matschik** [comment]
Thanks ! Don't forget to solve CI failing checks

**Pull request merged**